scripts/sct_propseg.py

```diff
@@ -437,22 +437,27 @@ def get_parser():
     sct.run(cmd, verbose)
 
     # extracting output filename
-    path_fname, file_fname, ext_fname = sct.extract_fname(input_filename)
-    output_filename = file_fname + "_seg" + ext_fname
+    file_seg = file_data + "_seg" + ext_data
```

Comment:
You can normalize fname_seg by: `fname_seg = os.path.abspath(os.path.join(folder_output, file_data + '_seg' + ext_data))`